recipes-support/salt/salt_3000.2.bb

```diff
@@ -188,3 +188,13 @@ FILES:${PN}-tests = "${PYTHON_SITEPACKAGES_DIR}/salt-tests/tests/"
 RDEPENDS:${PN}-ptest += "salt-tests python3-distro python3-mock"
 
 FILES:${PN}-bash-completion = "${sysconfdir}/bash_completion.d/${PN}-common"
+
```

Comment:
The github.com/ni/salt commit 4c849d71425a69bacdcff2b2d4398896983ececc also lists `CVE-2020-28243` as fixed. We should probably add it to the list here as it still shows as unpatched in the cve check output.